### ISSUE TITLE
adding parallax effect 

### ISSUE THREAD

**Issue by mridulabatra** [Algo-Phantoms/Algo-Phantoms-Frontend#71]
adding a parallax effect to the background  will built a beautiful user experience

**Comment by mridulabatra** [comment]
Please assign me this task as a part of GSSoC'21

**Comment by Shruti3004** [comment]
@mridulabatra go ahead

**Comment by ambivert-sam** [comment]
Hey!! @mridulabatra, I am a participant of GSSoC'21. I would take up this issue. 
Can you please assign me this issue.

**Comment by Shraddha13Jain** [comment]
Hey!! @mridulabatra, I am a participant of GSSoC'21. I would take up this issue.
Can you please assign me this issue.

**Comment by ravikr126** [comment]
are you completed @mridulabatra    ???

**Comment by mridulabatra** [comment]
yes sir I have made the pull request please check sorry for the delay @ravikr126
